packages/postgres-queries-builder/src/queries/remove/BatchRemove.ts

```diff
@@ -23,10 +23,9 @@ export const batchRemove = ({
     const selector = [].concat(primaryKey);
     const idSanitizer = sanitizeIdentifier(selector);
 
-    return ids => {
-        const cleanIds = ids.map(id => idSanitizer(id));
+    return (ids: Array<string | number | StringMap>) => {
```

Comment:
This typing is already declared in QueryFunction

Reply:
Yes, I know, but it wasn't enough ....  Transpilation failed without that.

Reply:
Have you tried to put Array<string | number | StringMap> typing in QueryFunction def ?

Reply:
Hey, it's working ! Les voies du TypeScript sont encore impénétrables pour moi :(